packages/terra-application-links/src/tabs/_Tab.jsx

```diff
@@ -2,12 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
-import { Route, matchPath } from 'react-router-dom';
+import { matchPath } from 'react-router-dom';
+import TabUtils from './_TabUtils';
 import styles from './ApplicationTabs.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * The history as provided by the `withRouter()` HOC.
+   */
+  history: PropTypes.object.isRequired,
```

Comment:
If this is provided by the HOC, do we need to mark it required?

Reply:
It doesn't hurt to be descriptive of the api, even if it's an internal component.